QuickStart.md

```diff
@@ -0,0 +1,41 @@
+# Development Environment 
+
+Get a local development environment up and running with as few steps as possible.
+
+## Required Variables
+
+- **OPENAI_API_KEY:** [Get your GPT-4 OpenAI key](https://platform.openai.com/api-keys)
+
+## Start
+
+```shell
+yarn set version classic
+yarn install
+yarn contracts:solo-up
+yarn contracts:deploy:solo
+
+PORT=3000 \
+ ORIGIN="*" \
+ OPENAI_API_KEY="sk-proj-.." \
+ ADMIN_MNEMONIC="denial kitchen pet squirrel other broom bar gas better priority spoil cross" \
+ NETWORK_URL=http://localhost:8669 \
+ REWARD_AMOUNT=1 \
+ yarn dev
```

Comment:
Like the idea of the quickstart! But 

`OPENAI_API_KEY="sk-proj-.."` would configure the backend with a wrong API key

Users might mistake the steps as a copy and paste and everything will work.

Possible solution could be adding the "Result" section of the Quickstart below the Readme here:
![image](https://github.com/user-attachments/assets/7a5d4021-51cb-4fc8-bde6-9e4170075653)